### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by geraldgsh**
1. Prepare the directory structure
2. Implement the App component
3. Implement the BooksList component
4. Implement the Book component
5. Implement the BooksForm component
6. Define the Redux actions and reducers
7. Initialize the Redux store

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <BookList />
+    <BooksForm />
+  </div>
+);
+
+export default App;
```
Comment (shloch) [inline src/App.js:14]: pls relocate this file to `src/components` folder

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Books';
+
```
Comment (shloch) [inline src/components/BookList.js:5]: pls relocate this file into `src/containers` folder
Screen Shot 2020-04-22 at 8 17 04 AM

File: src/components/BooksForm.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
```
Comment (shloch) [inline src/components/BooksForm.js:2]: pls relocate this file into `src/containers` folder
Screen Shot 2020-04-22 at 8 17 04 AM

**Review by shloch — APPROVED** [review]
## Hello @geraldgsh , @Aaron-RN 

Perfect execution...You get a thumbs up, but please do me a favour below before merging.
Can you please relocate few files mentioned below ??

Everything technical, you guys nailed it........
Unto next milestone
accomplished

### HAPPY CODING

**Comment by geraldgsh** [comment]
Hello @shloch. Thanks for reviewing our work. We make suggest amendments in our approved milestone 3.

File: src/App.js (hunk 1 above)
Comment (geraldgsh) [reply src/App.js:14]: Will be resolved in Milestone 3 to avoid misalignment of code

File: src/components/BookList.js (hunk 2 above)
Comment (geraldgsh) [reply src/components/BookList.js:5]: Will be resolved in Milestone 3 to avoid misalignment of code

File: src/components/BooksForm.js (hunk 3 above)
Comment (geraldgsh) [reply src/components/BooksForm.js:2]: Will be resolved in Milestone 3 to avoid misalignment of code

**Pull request merged**